tests/functional/cylc-cat-log/14-auto-mode.t

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test "cylc cat-log" using auto mode with a custom tail /command
+. "$(dirname "$0")/test_header"
+#-------------------------------------------------------------------------------
+set_test_number 4
+install_workflow "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
+create_test_global_config "" "
+[platforms]
+   [[localhost]]
+        tail command template = $PWD/bin/my-tailer.sh %(filename)s
+"
+#-------------------------------------------------------------------------------
+TEST_NAME="${TEST_NAME_BASE}-validate"
+run_ok "${TEST_NAME}" cylc validate "${WORKFLOW_NAME}"
+workflow_run_ok "${TEST_NAME_BASE}-run" cylc play "${WORKFLOW_NAME}"
+cylc workflow-state "${WORKFLOW_NAME}//1/foo:started" --interval=1
+sleep 3
```

Comment:
The `cylc workflow-state` command should render this unncessary:

```suggestion
```

Reply:
Perhaps you need to test for `:finished`?

Reply:
I wonder (untested) if adding a cylc message to the start of the `script` and registering that output:

```diff
        script = """
            cylc message -- "If we invade, all Laconia will be destroyed"
            for I in $(seq 1 100); do
                echo "from $CYLC_TASK_NAME $I"
                sleep 1
            done
        """
        [[[outputs]]]
            laconic = "If we invade, all Laconia will be destroyed"
```

would allow you to ensure that the task has _really_ started?

Reply:
Agreed, but suggest that someone with a bit more experience with these functional tests attempts to solve the issue.
So, suggest this is OOS for little old me.